src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineGroupConcatFieldDescriptor.php

```diff
@@ -61,9 +71,15 @@ public function getSelect()
     }
 
     /**
-     * Returns all the joins required for this field.
-     *
-     * @return DoctrineJoinDescriptor[]
+     * {@inheritdoc}
+     */
+    public function getSimpleSelect()
```

Comment:
Couldn't that break some queries? What if the group concat field isn't in the select list, but still used to filter stuff?